usagov/vote-gov-nvrf-app
# Vote-2900: Remove password page from app

Ticket: [Vote-2900](https://cm-jira.usa.gov/browse/VOTE-2900)

Description:  Removing the login for the NVRF app and updating the cypress tests 

Testing:
1. start up the app and verify that user is not prompted to enter login information. 
2. fill out form and ensure no major regressions
3. spilt terminal and run `npm run cy:test` and verify that cypress tests are still passing
